mex/model/entities/organization.json

```diff
@@ -151,10 +161,12 @@
             "description": "Identifier from Wikidata.",
             "items": {
                 "examples": [
-                    "http://www.wikidata.org/entity/Q679041"
+                    "http://www.wikidata.org/entity/Q679041",
+                    "https://www.wikidata.org/wiki/Q918501",
+                    "https://www.wikidata.org/wiki/Q491566"
```

Comment:
`https://www.wikidata.org/wiki/Q` is formally NOT the correct URI of a wikidata item. URIs must conatin `entity`. I would not like to include the wrong URI pattern in the examples. And also it would not match with the pattern, am I right?

Reply:
absolutely right! i will fix that